lua/ttt2/libraries/migrations.lua

```diff
@@ -0,0 +1,62 @@
+---
+-- A TTT2 version migrations library
+-- @author ZenBre4ker
+-- @module migrations
+
+if SERVER then
+	AddCSLuaFile()
+end
+
+migrations = {}
+migrations.folderPath = "terrortown/migrations/"
+migrations.databaseName = "ttt2_migrations"
+migrations.savingKeys = {
+	executedAt = {
+		typ = "string",
+		default = ""
+	}
+}
+
+sql.CreateSqlTable(migrations.databaseName, migrations.savingKeys)
+migrations.orm = orm.Make(migrations.databaseName)
+
+---
+-- Runs all missing forward gamemode migrations.
+-- @realm shared
+function migrations.Apply()
+	local files = file.Find(migrations.folderPath .. "*.lua", "LUA", "nameasc") or {}
+
+	local fileExecutionCounter = 0
+	for i = 1, #files do
+		local fileName = files[i]
+		local fullPath = migrations.folderPath .. files[i]
+
+		local fileInfo = migrations.orm:Find(fileName)
+
+		if fileInfo then continue end
+
+		MsgN("[TTT2] Migrating: ", fileName)
```

Comment:
@Histalek What's our policy now on those messages?

I think we need to write that down somewhere.

Reply:
This should be `Dev(1, xx)`